File: Darts/src/burlton/dartzee/code/screen/AboutDialogDarts.java
package burlton.dartzee.code.screen;

import java.awt.Window;

import burlton.desktopcore.code.screen.AbstractAboutDialog;
import burlton.core.code.util.OnlineConstants;

public class AboutDialogDarts extends AbstractAboutDialog
{
	@Override
	public String getProductDesc()
	{
		return "Darts " + OnlineConstants.DARTS_VERSION_NUMBER;
	}

	@Override
	public Window getChangeLog()
	{
		return new ChangeLog();
	}

}

File: Darts/src/burlton/dartzee/code/screen/AboutDialogDarts.kt
package burlton.dartzee.code.screen

import burlton.dartzee.code.utils.DARTS_VERSION_NUMBER
import burlton.dartzee.code.core.screen.AbstractAboutDialog

class AboutDialogDarts : AbstractAboutDialog()
{
    override fun getProductDesc() = "Darts $DARTS_VERSION_NUMBER"
    override fun getChangeLog() = ChangeLog()
}
